spark-cassandra-connector/src/main/scala/org/apache/spark/sql/cassandra/ConfCache.scala

```diff
@@ -0,0 +1,133 @@
+package org.apache.spark.sql.cassandra
+
+import com.datastax.spark.connector.cql.CassandraConnectorConf
+import com.datastax.spark.connector.rdd.ReadConf
+import com.datastax.spark.connector.writer.WriteConf
+import org.apache.commons.lang.StringUtils
+
+import scala.collection.mutable
+
+
+object ConfCache {
```

Comment:
It caches the user defined configurations, so it can be used by all the following sqlContext. sql queries. It can be updated by user.